src/components/MenuComponents/ViewMenuItem/ViewMenuItem.module.css

```diff
@@ -124,8 +124,8 @@
 }
 
 .title img {
-  width: 375px;
-  height: 375px;
+  width: 100%;
+  height: 100%;
```

Comment:
@thomasdahlberg each menu item won't have a fixed layout ratio if we use 100%. I think it's better to use a fixed pixel for the desktop and the tablet view. The downside is the image section won't be optimized with wider images. I'm still learning CSS so let me know if you have a better suggestion!

100%
![100](https://user-images.githubusercontent.com/51396212/113045891-38bb2000-9154-11eb-9193-e7734f3dce56.png)

330px (The default menu item image has 330px x 330px when it's 100%)
![fixed](https://user-images.githubusercontent.com/51396212/113046135-86378d00-9154-11eb-8d8c-fc21733babc1.png)